### PR TITLE
@damassi => IE Fixes for Video Page

### PR DESCRIPTION
**Pull request by kanaabe**
Fixes IE issues -- About sections without widths, -ms prefix for scrubber, etc. Looks like Prettier had a go with these files, I don't know remember turning that back on but 🤷‍♀️ I'll note the actual changes inline.

### CODE REVIEW

File: src/Components/Publishing/Nav/Nav.tsx [hunk 1]
```diff
@@ -41,39 +41,21 @@ export class NavComponent extends React.Component<Props, State> {
...
+    const { sponsor, className, canFix, transparent, title } = this.props
     const { isFixed } = this.state
 
     return (
       <div>
-        <NavContainer
-          transparent={!isFixed && transparent}
-          className={className}
-          isFixed={canFix && isFixed}
-        >
```
Comment (damassi) [inline src/Components/Publishing/Nav/Nav.tsx]: 😢 bye bye readable code

Comment (damassi) [reply src/Components/Publishing/Nav/Nav.tsx]: 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢

File: src/Components/Publishing/Nav/Nav.tsx [hunk 2]
```diff
@@ -41,39 +41,21 @@ export class NavComponent extends React.Component<Props, State> {
...
-              ? title
-              : <a href="/magazine">Artsy Editorial</a>
-            }
-          </Title>
+          <Title>{title ? title : <a href="/magazine">Artsy Editorial</a>}</Title>
         </NavContainer>
-        <Waypoint
-          onEnter={() => this.setPosition(false)}
-          onLeave={() => this.setPosition(true)}
-        />
```
Comment (damassi) [inline src/Components/Publishing/Nav/Nav.tsx]: Bye 😢

File: src/Components/Publishing/Video/VideoAbout.tsx [hunk 3]
```diff
@@ -19,46 +19,31 @@ interface TitleProps {
...
     return (
       <VideoAboutContainer>
         <Col xs={12} sm={4}>
-          <Title color={color}>
-            Credits
-          </Title>
-          {editCredits
-            ? <Text layout="standard">{editCredits}</Text>
-            : <Text layout="standard" html={media.credits} />
-          }
```
Comment (damassi) [inline src/Components/Publishing/Video/VideoAbout.tsx]: Bye 😢

File: src/Components/Publishing/Video/VideoAbout.tsx [hunk 4]
```diff
@@ -19,46 +19,31 @@ interface TitleProps {
...
-            About the Film
-          </Title>
-          {editDescription
-            ? <Text color={color} layout='standard'>{editDescription}</Text>
-            : <Text color={color} layout="standard" html={media.description} />
-          }
-          <ShareDate
-            color={color}
-            article={article}
-          />
```
Comment (damassi) [inline src/Components/Publishing/Video/VideoAbout.tsx]: I'm forking prettier

File: src/Components/Publishing/Nav/Nav.tsx (hunk 1 above)
Comment (orta) [reply src/Components/Publishing/Nav/Nav.tsx]: I don't really get this, the new code looks better to me. 

It's short enough to fit on a single line, so it becomes more readable by being read left to right?

Comment (kanaabe) [reply src/Components/Publishing/Nav/Nav.tsx]: Yeah I guess it’s just personal preference - i like Prettier for everything except being able to do multiline components like this. We do this consistently in the Publishing directory and in Positron so I think we’re just used to it. But I can see how this is troublesome within the repo for half of us to use it and the other half not if we want to share components for instance!

Comment (orta) [reply src/Components/Publishing/Nav/Nav.tsx]: That feels like the answer is to get prettier installed and set up the same in Positron too 👍

**Pull request closed without merging**